packages/block-editor/src/utils/object.js

```diff
@@ -65,3 +65,17 @@ export function uniqByProperty( array, property ) {
 		return seen.has( value ) ? false : seen.add( value );
 	} );
 }
+
+/**
+ * Helper to omit keys from a given object.
+ *
+ * @param {Object}   object Object.
+ * @param {string[]} keys   Keys to omit.
+ *
+ * @return {Object} Object without the omitted keys.
+ */
+export function omit( object, keys ) {
```

Comment:
See my comment above for introducing a utility function for omitting properties. It can be helpful if the properties are with dynamic names, but if they're static (like the `key` instance above) I think we should just do [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) with rest parameters.

Reply:
Note that the utility already existed in the package (in a specific file) and was also being used with static properties. 

I'm hesitant, I think "omit" can be a bit better than leaving unused variables.

Reply:
I consider the utility quite [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) and an unnecessary abstraction unless working with dynamic object keys, but I have no strong feelings if you really prefer it that way.

Reply:
I removed the utility 👍